eventmachine.gemspec

```diff
@@ -12,7 +12,12 @@ Gem::Specification.new do |s|
   s.files = `git ls-files README.md CHANGELOG.md GNU LICENSE rakelib ext java lib docs`.split
   s.test_files = `git ls-files tests examples`.split
 
-  s.extensions = ["ext/extconf.rb", "ext/fastfilereader/extconf.rb"]
+  if RUBY_PLATFORM =~ /java/
+    s.files << "lib/rubyeventmachine.jar"
+    s.platform = 'java'
+  else
+    s.extensions = ["ext/extconf.rb", "ext/fastfilereader/extconf.rb"]
+  end
```

Comment:
I had been using `rake gem` and `rake package` and they didn't seem to be java-specific (it was still trying to build c-extensions when I installed the local one), but I didn't try `rake build`. Also, entirely possible I was doing something dumb.

I'd be happy to pull that out. I found a few more things I'd like to contribute. Would you prefer those in a separate PR, or in this same one?